README.md

```diff
@@ -1,44 +1,45 @@
-# Constructor-IO Python Client [![Build](https://travis-ci.org/Constructor-io/constructorio-python.svg?branch=master)](https://travis-ci.org/Constructor-io/constructorio-python) [![Version](https://img.shields.io/pypi/v/constructor-io.svg)](https://pypi.python.org/pypi/constructor-io) [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
+# Constructor.io Python Client
 
+[![Version](https://img.shields.io/pypi/v/constructor-io.svg)](https://pypi.python.org/pypi/constructor-io)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT)
 
+A Python client for [Constructor.io](http://constructor.io/). [Constructor.io](http://constructor.io/) provides search as a service that optimizes results using artificial intelligence (including natural language processing, re-ranking to optimize for conversions, and user personalization).
 
-A Python package for the [Constructor.io API](http://constructor.io/docs).  Constructor.io provides a lightning-fast, typo-tolerant autocomplete service that ranks your users' queries by popularity to let them find what they're looking for as quickly as possible.
-
-## Installation
+## 1. Install
 
 ```
-pip install constructor-io
+pip install constructorio-python
```

Comment:
This should change if we are planning to keep the name

Reply:
Correcto - let's keep the name